cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -197,3 +161,27 @@ the nodes of any clusters where {CNVProductName} is deployed. The nodes might no
 drain if there are virtual machines running on top of them.
 The current solution is to put nodes into maintenance.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1707427[*BZ#1707427*])
+
+* When running `virtctl image-upload` to upload large VM disk images in `qcow2`
+format, an end-of-file (EOF) error may be reported after the data is
+transmitted, even though the upload is either progressing normally or completed.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1754920[*BZ#1754920*])
++
+Run the following command to check the status of an upload on a given PVC:
++
+----
+$ oc describe pvc <pvc-name> | grep cdi.kubevirt.io/storage.pod.phase
+----
+
+* When attempting to create a virtual machine on hardware with a Haswell CPU,
+the virtual machine creation fails and `libvert` declares
+`<model usable='yes'>Haswell</model>`. This is a change in behavior from
+previous verions of {CNVProductName}, where `libvirt` declared
+`<model usable='no'>Haswell</model>`.
++
+As a workaround, select a different CPU model, if possible, or run the following 
+command to fix the incorrect flag on the Haswell CPU:
++
+----
+$ oc label node $THE_NODE_NAME feature.node.kubernetes.io/cpu-model-Haswell=false --overwrite
```

Comment:
@bgaydosrh can you double check with Karel Simon about this? Wouldn't the node labeler overwrite this on the next time it is running?

Reply:
Node-labeller overwrites labels when it is restarted and after that there will be again  `cpu-model-Haswell=true`

Reply:
Dropping workaround per @ksimon1. cc: @dankenigsberg

Reply:
Just checking to verify if I can still say: "As a workaround, select a different CPU model, if possible." 

I know it's not great, but it's something. @ksimon1 @dankenigsberg

Reply:
Yes. Let's keep the last recommendation.

Reply:
Done.